### PR TITLE
set functions: add, remove, size, isEmpty, contains, and union workin…

### PR DESCRIPTION
**Pull request by TrevorJamesH**
…g with test

### CODE REVIEW

File: src/set.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  }
+
+
+  size(){
+    return this.index
+  }
+
+
+  isEmpty() {
+    if(this.index === 0){
```
Comment (shakalee14) [inline src/set.js]: i'm pretty sure you can return a single line that will evaluate for truthiness

``` return this.index === 0 ```

File: src/set.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    else{
+      return false
+    }
+  }
+
+  remove(element){
+    for(let i = 0; i < this.index; i++){
+      if(this.storage[i] === element){
+        this.storage.slice(i)
+        //refactor slice!!!!
```
Comment (shakalee14) [inline src/set.js]: take out comments

File: src/set.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+  add(element){
+    if(!this.contains(element)){
+      this.storage[this.index++] = element
+    }
+  }
+
+  contains(element){
+    for(let i = 0; i < this.index; i++){
+      if(this.storage[i] === element){
```
Comment (shakalee14) [inline src/set.js:17]: practice using a ternary here

File: src/set.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        this.storage.slice(i)
+        //refactor slice!!!!
+        this.index--
+      }
+    }
+  }
+
+  union(element){
+    for(let i = 0; i < element.length; i++){
+    //refactor .length!!!
```
Comment (shakalee14) [inline src/set.js]: take out comment here

File: spec/set.js [hunk 5]
```diff
@@ -0,0 +1,104 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import Set from '../src/set'
+
+chai.use(chaiChange)
+
+describe.only('Set', () => {
```
Comment (shakalee14) [inline spec/set.js:7]: may want to take of .only when you merge to master

**Review by shakalee14 — COMMENTED** [review]
few suggestions

File: src/set.js (hunk 3 above)
Comment (TrevorJamesH) [reply src/set.js:17]: Thanks for the feedback, how would a ternary work within the loop? I only want it to return false if its gone through the whole loop comparing the contents of this.storage to the element.

**Pull request merged**